src/site/_includes/layouts/_base.njk

```diff
@@ -48,6 +48,19 @@
   <link rel="mask-icon" href="/assets/icon-safari-mask.svg" color="#532A45">
   <link rel="manifest" href="/manifest.json">
   <link rel="stylesheet" href="{{ "src/site/index.scss" | inputToRevvedOutput }}">
+  {% if vars.trackingId %}
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    window.trackingId = '{{ vars.trackingId }}'
+    function gtag(){dataLayer.push(arguments);}
+    gtag('consent', 'default', {
+      'analytics_storage': 'denied'
```

Comment:
Here we set the default `analytics_storage` to denied so it doesn't install any cookies